lib/fpm/package/virtualenv.rb

```diff
@@ -25,6 +25,10 @@ class FPM::Package::Virtualenv < FPM::Package
   option "--other-files-dir", "DIRECTORY", "Optionally, the contents of the " \
   "specified directory may be added to the package. This is useful if the " \
   "virtualenv needs configuration files, etc.", :default => nil
+  option "--pypi-extra", "PYPI_EXTRA_URL",
```

Comment:
Curious, Is this a good name for the flag? Would it be more descriptive as --pypi-extra-url ? Thoughts? :)